crates/red_knot_module_resolver/src/resolver.rs

```diff
@@ -386,6 +402,8 @@ pub(crate) struct ModuleResolutionSettings {
     /// Note that `site-packages` *is included* as a search path in this sequence,
     /// but it is also stored separately so that we're able to find editable installs later.
```

Comment:
Orthogonal to this PR, but this comment is confusing to me: where is it "also stored separately"?

Reply:
It's not anymore after Micha refactored the code in https://github.com/astral-sh/ruff/commit/91338ae9021d64ba39d82717d633a510d70649f7 :-)

The comment is out of date following that commit

Reply:
Originally in https://github.com/astral-sh/ruff/commit/9a2dafb43d36b62d45a604dea58224a0884cd6e5, the `site-packages` search path was stored as a separate field on the struct as well as being present in the `static_search_paths` field; that's what this comment referred to, before the code was refactored